eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/BinaryPayload.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 - 2016 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.eventuate
+
+import com.google.protobuf.ByteString
+
+/**
+ * Represents a serialized payload.
+ * @param bytes serialized representation of a generic payload
```

Comment:
just *serialized payload*? Especially, *generic* confuses me and *representation* is the actually the `BinaryPayload`, isn't it?